en/reference/assertions.md

```diff
@@ -29,58 +32,612 @@ sign-key-sha3-384: <key id> # Encoded key id of signing key
 
 Every assertion will have a `type` and `sign-key-sha3-384`, as well as a signature, and most will have an `authority-id`. Values may be scalars (strings, integers, booleans), lists, or maps. Some of the headers play the role of an index, which uniquely specifies the context of this assertion given the type. Most assertions will also have a revision which enables a particular assertion to be updated by issuing another assertion of the same type and index with a higher revision. Given a particular type and index, there is only one “latest” valid assertion that properly determines policy for a system - the one with the highest revision. For a given assertion the index headers must all be defined.
 
-The following assertions are currently supported:
+Some headers contain identifiers of one type or another. These ids are formed by
+a sequence of 32 random characters taken out of an alphabet of 62 characters
+(uppercase letters + lowercase letters + numbers). However, in some cases the
+identifier is a fixed name like "canonical". This only happens in special cases.
+
+When headers are defined as UTC date/time, the format that is needed is the same as the
+one generated by the command.
+```
+date -Iseconds --utc
+```
+
+It can be instructive to learn about the assertions that are stored in a system.
+You can find out with the command
+```
+snap known <assertion type>
+```
+
+Additionally, you can download the assertions associated to snaps in the store
+with the command
+```
+snap download <snap name>
+```
+which will download the snap and a file with its assertions. The file will
+contain account-key of the snap owner, snap-declaration, and snap-revision assertions.
+
+Below we define the currently supported assertions. Unless otherwise noted, all
+fields are madatory.
 
 ## account
```

Comment:
I think it would make sense to have a page per assertion to not make this a pretty long page the user scrolls and scrolls. What do you think?